Formula/weevely.rb

```diff
@@ -0,0 +1,61 @@
+class Weevely < Formula
+  desc "Weaponized web shell"
+  homepage "https://github.com/epinna/weevely3"
+  url "https://github.com/epinna/weevely3/archive/v3.5.tar.gz"
+  sha256 "edb285469b2c8504797c1a35e30d728ddd88b56de37376c4464081c4d1cdbe9e"
+
+  depends_on "python@2"
```

Comment:
Does it not work with macOS system python?

Reply:
Yes, it does. I didn't know that it was recommended to use system python.

Reply:
Fixed.